RTN-004.tex

```diff
@@ -145,263 +177,257 @@ \subsubsection{Benefits}\label{sssec:intro_del_bene}
-The application form will remain open, accumulating submissions, for the duration of DP0.
-Applicants will not need to resubmit their application for DP0.2, as all applications received by that time would be considered.
-
-To ensure that at least some accounts are available for new people to join for DP0.2, ~25\% of the 300 total RSP accounts will not be allocated until the second stage (DP0.2).
-The DP0.1 data will be available to delegates who join with the second stage.
-The following paragraph provides additional motivation for this multi-stage approach to accounts allocation.
-
-The entire duration of DP0 is long, at least 1 year and potentially longer.
-It is anticipated that many delegates would not be able or willing to make time commitment for DP0.1 (Section \ref{sssec:intro_del_resp}).
-Furthermore, some scientists might be more interested in the DP0.2 data set than DP0.1.
-Additionally, achieving diverse and equitable access to DP0 might also take some time: the advertisements must percolate, people must free up time in their schedules or obtain grants to fund their work, and the DP0-related resources and support network will become enriched over time (and thus more attractive to a wider variety of participants).
-To avoid the perception that access to any of DP0 is restricted to individuals who can commit to participating at the time of DP0.1, 25\% of the 300 RSP accounts will be reserved for DP0.2.
-Any accounts relinquished by DP0.1 participants will be added to this reservation and reallocated for DP0.2.
+These dates are provided in Table \ref{tab:selection_timeline}.
+As described in Section \ref{ssec:intro_del}, delegates need not begin their DP0.1 participation at the time of account activation, and may distribute their time over DP0 at their convenience.
 
 \begin{table}[!h]
 \centering
 \caption{Applicants who submit by the dates in column 3 will be notified by the dates in column 4.}\label{tab:selection_timeline}
 \begin{tabular}{lllll}
 \hline
-Stage & \% (of 300) & Applications Due & Delegates Notificatied & Data Available \\
+Stage & Applications Due & Delegates Notified & Data Available \\
 \hline \hline
-DP0.1 & 75\%    &  Apr 30 2021   &  May 31 2021  &  Jun 30 2021 \\
-DP0.2 & 100\%  &  Jan 31 2022   &  Feb 28 2022  &  Mar 31 2022 \\
+DP0.1 & Apr 30 2021   &  May 31 2021  &  Jun 30 2021 \\
+DP0.2 & Apr 30 2022   &  May 31 2022  &  Jun 30 2022 \\
 \hline
 \end{tabular}
 \end{table}
 
 
-\section{Resources for DP0 Delegates}\label{sec:res}
+\subsection{DP0.1 Allocations}\label{ssec:sel_dp01}
+
+When the DP0.1 selection algorithm was run in May 2021, the total number of DP0 accounts was going to be 300 (not 600) and 25\% of them were reserved for DP0.2.
```

Comment:
was going to be -> was set at 300 and 25% ....

Reply:
updated